client/api/src/main/java/io/smallrye/graphql/client/mpapi/Error.java

```diff
@@ -0,0 +1,15 @@
+package io.smallrye.graphql.client.mpapi;
```

Comment:
I think we should stick with
`org.eclipse.microprofile.graphql.client` -> `io.smallrye.graphql.client`

So the idea is that at some point, things in smallrye api should move to (or release in) MP. Then the only change should be `org.eclipse.microprofile` -> `io.smallrye`

Reply:
Ok, renamed it